doc/source/templates/01_batch_inference/README.md

```diff
@@ -1,28 +1,14 @@
 # Scaling Batch Inference with Ray Data
 
-This template is a quickstart to using [Ray
-Data](https://docs.ray.io/en/latest/data/data.html) for batch
-inference. Ray Data is one of many libraries under the [Ray AI
-Runtime](https://docs.ray.io/en/latest/ray-air/getting-started.html).
-See [this blog
-post](https://www.anyscale.com/blog/model-batch-inference-in-ray-actors-actorpool-and-datasets)
-for more information on why and how you should perform batch inference
-with Ray!
+| Template Specification | Description |
+| ---------------------- | ----------- |
+| Summary | This template walks through GPU batch inference on an image dataset using a PyTorch ResNet model. |
+| Time to Run | Less than 2 minutes to compute predictions on the dataset. |
+| Minimum Compute Requirements | No hard requirements. The default is 4 nodes, each with 1 NVIDIA T4 GPU. |
+| Cluster Environment | This template uses the latest Anyscale-provided Ray ML image using Python 3.9: [`anyscale/ray-ml:2.4.0-py39-gpu`](https://docs.anyscale.com/reference/base-images/ray-240/py39#ray-ml-2-4-0-py39). If you want to change to a different cluster environment, make sure that it is based off of this image! |
```

Comment:
If we are hardcoding Ray 2.4.0 here, do we have a procedure to update this when a new version or Ray gets released? (like the imminent Ray 2.5)?

Reply:
Currently it's a manual maintenance process. I do really want these to all just be able to use a `latest` alias, but this alias does not exist at the moment.

The anyscale default cluster environments all have a `build_id`, which is what we supply in the templates yaml file: `anyscaleray-ml240-py39-gpu`

There is no `anyscaleray-mllatest-py39-gpu`. The `anyscale/ray:latest-py39-gpu` does exist though. @sofianhnaide Who should I talk to for this alias to be added?

Reply:
The images for BYOD will have to be updated per-release as well. This could be automated if we're able to use the buildfarm to create the image every release.

Currently, it's not possible to create a cluster environment and extract out the image, so I had to manually create the docker image.